src/azure-cli/azure/cli/command_modules/containerapp/_utils.py

```diff
@@ -1477,12 +1479,13 @@ def load_cert_file(file_path, cert_password=None):
             elif os.path.splitext(file_path)[1] in ['.pfx']:
                 cert_data = f.read()
                 try:
-                    p12 = crypto.load_pkcs12(cert_data, cert_password)
+                    # The password to use to decrypt the data. None if the PKCS12 is not encrypted.
+                    cert_password_bytes = cert_password.encode('utf-8') if cert_password is not None else None
```

Comment:
what if the cert_password is ""

Reply:
Add test for password is "" in `test_load_cert_file`.

Reply:
If Password is "", it means the PKCS12 is not encrypted.